protocol/src/main/protobuf/UserBitShared.proto

```diff
@@ -320,3 +320,17 @@ message Jar {
   repeated string function_signature = 2;
 }
 
+
+/* Session parameter
+message SessionParam {
+  optional string key = 1;
+  optional string value = 2;
+}
+*/
+
+/* User session context that has session id and session parameters
+*/
+message UserSessionContextInformation {
+  optional string session_id = 1;
+  // repeated SessionParam session_params = 2
+}
```

Comment:
Why we are using protobuf for UserSessionContextInformation and not regular class?